tests/bettertransformer/test_bettertransformer_vision.py

```diff
@@ -133,3 +135,32 @@ def test_raise_autocast(self, test_name: str, model_id, padding, max_length=20):
     )
     def test_raise_train(self, test_name: str, model_id, padding, max_length=20):
         super().test_raise_train([model_id], padding=padding, max_length=max_length)
+
+    @parameterized.expand([(True,), (False,)])
+    def test_invert_model_logits(self, keep_original_model=True, **preprocessor_kwargs):
+        r"""
+        Test that the inverse converted model and hf model have the same logits
+        """
+        # The first row of the attention mask needs to be all ones -> check: https://github.com/pytorch/pytorch/blob/19171a21ee8a9cc1a811ac46d3abd975f0b6fc3b/test/test_nn.py#L5283
+        for model in self.all_models_to_test:
```

Comment:
same comment

Reply:
fixed in [c5179e3](https://github.com/huggingface/optimum/pull/676/commits/c5179e396f023461fd7b11227d8d6121c0f7c2a7)